configure/lang/go/README.md

```diff
@@ -0,0 +1,76 @@
+# lang-go
+
+This folder contains the deployment manifests for the [Go language extension](https://sourcegraph.com/extensions/sourcegraph/go). 
+
+## Installation instructions
+
+### Setup TLS/SSL 
+
+TLS/SSL is required for secure communication with the language server. Once you have completed ["Configure TLS/SSL"](../../../docs/configure.md#configure-tlsssl) in [docs/configure.md](../../../docs/configure.md#configure-tlsssl), change the `sourcegraph.example.com` domain in [configure/lang/go/lang-go.Ingress.yaml](lang-go.Ingress.yaml) to the real domain name that you are using for your Sourcegraph instance.
+
+```diff
+spec:
+    tls:
+    - hosts:
+    # 🚨 TLS is required for secure communication with the language server. 
+    # See the customization guide (../../../docs/configure.md) for information
+    # about configuring TLS
+-     - sourcegraph.example.com
```

Comment:
Why not keep "sourcegraph.example.com"? That way someone can replace all instances of it at the same time?